website/docs/error-reference.md

```diff
@@ -702,6 +702,68 @@ you'll get this error—the latter export is redundant, as it's implied by expor
 
 To fix this error, simply remove the more specific export.
 
+## 3717
+
+> This error is specific to Stripe's custom `--stripe-packages` definition. If
+> you are at Stripe, please see [go/modularity](http://go/modularity) for more.
```

Comment:
lol, i have no idea what i meant by "`--stripe-packages` definition"

I'm going to change this to "`--stripe-packages` mode" after you land this change.